### ISSUE TITLE
Newbie - Just download the zip and diskc and libstdc++.dylib missing

### ISSUE THREAD

**Issue by kwccoin** [litchie/dospad#24]
Hi,

Is there any setting or any file to copied before I can compile?  The Xcode 6.4 (for iOS 9) mark
- disk c
- libstdc++.dylib

For your kind advice.

**Comment by litchie** [comment]
diskc is a folder to hold bundled games, all those files will be copied to Documents folder on launch. libstdc++.dylib is in red, but should not be a problem.

**Comment by DennisNg** [comment]
Thanks for the quick response.  

I add the diskc and it seem the build is fine but there is no launch of any program.  Need a bit more mobile, I switch to my newer mac air, take apple advice to do the setting change and it seems now libstdc++.6.0.9.dylib, libstdc++.dylib, lib.dylib, libc.dylib all not there.  But the build is completed without any error (3xx warning).

The Products folder said it has built libSDLiPhoneOS.a  but not dosed.app libdosbox.a libminizip.a libSDL_net.a iDOS.app. 

Not sure what I can do next.  BTW, I did release one app on the store and hence not that bad.    Not totally useless but probably not get this kind of work.

I did get the iDOS1 and now iDOS2.  Just want to know how it works.  And also run some old 8086 assembler on the portable DOS machine aka iPAD. 

Best regards and hope not wasting your time.

Dennis
